src/app/components/page-not-found/page-not-found.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="content">
+  <div class="sticky">
+    <div class="error-container">
+      <img src="assets/img/sticker404.svg" class="sticker-1">
```

Comment:
co to jest ? Jak każdy sticker-x jest tym pomarańczowym prostokatem z slowem 'oops' na napisie z h1 404 to wg mnie jest to przerost formy nad treścią. Nawet w Paint idzie takie coś zrobić jako jednej obrazek i załączyć. Nie wspomnę o milionie przykładów fotek stylowanych z 404. Inwencja twórcza w programowaniu jest wskazana ale bez przesady (tym bardziej, że pewnie jakiś czas spędziłeś na odpowiednie trafienie zdjęciem w dobrym miejscu na liczbie 404)

Reply:
a gdzie jest plik sticker404.svg ogolnie ?